src/khepri_machine.erl

```diff
@@ -1252,16 +1255,30 @@ handle_aux(
               [restore_projection(Projection, Tree, PathPattern) ||
                Projection <- Projections]
       end),
-    {no_reply, AuxState, LogState};
+    {no_reply, AuxState, IntState};
 handle_aux(
   _RaState, cast,
   #restore_projection{projection = Projection, pattern = PathPattern},
-  AuxState, LogState, State) ->
+  AuxState, IntState) ->
+    State = ra_aux:machine_state(IntState),
     Tree = get_tree(State),
     ok = restore_projection(Projection, Tree, PathPattern),
-    {no_reply, AuxState, LogState};
-handle_aux(_RaState, _Type, _Command, AuxState, LogState, _MachineState) ->
-    {no_reply, AuxState, LogState}.
+    {no_reply, AuxState, IntState};
+handle_aux(_RaState, cast, tick, AuxState, IntState) ->
```

Comment:
@kjnilsson should this clause match on the leader state? Otherwise will we append the new command once for every member of the cluster?

Reply:
The state machine version was not bumped with this patch. An old machine that doesn’t know `#expire_dedups{}` will log an error and reply with a `?khepri_exception()`. Is that ok?

Reply:
I thought it would be alright since I assumed followers drop the `mod_call` effect that logs an error but on second look followers execute those as well.

I have another patch coming that will need to bump the machine version anyways so I will group these changes together